node/build.rs

```diff
@@ -1,16 +1,46 @@
-use std::env;
+use std::{env, process::Command};
 
-use vergen::EmitBuilder;
+const NODE_BUILD_PROFILE_ENV_VAR: &str = "NODE_BUILD_PROFILE";
 
+const DEFAULT_BUILD_PROFILE: &str = "PROFILE";
+
+///
+/// `casper-node` build script to capture the git revision hash and export it to cargo to include
+/// it in the version information
+///
+/// Notes: This script exports information to cargo via println! with the old invocation prefix of
+/// `cargo:`, if/when the node uses a Rust version `1.77` or above, this should be changed to
+/// `cargo::` as the prefix changed in that version of rust
 fn main() {
-    if let Err(error) = EmitBuilder::builder().fail_on_error().git_sha(true).emit() {
-        println!("cargo:warning={}", error);
-        println!("cargo:warning=casper-node build version will not include git short hash");
-    }
+    match Command::new("git")
+        .arg("rev-parse")
+        .arg("--short")
```

Comment:
I think the existing code is using the full hash, based on vergen docs: https://docs.rs/vergen/latest/vergen/struct.EmitBuilder.html#method.git_sha, so this will slightly change the format

I can't think of anything that'd break because of this using shortened hash though, but I thought it might be worth pointing out